pkg/cloudprovider/providers/gce/gce_loadbalancer_external_test.go

```diff
@@ -40,21 +41,38 @@ import (
 )
 
 const (
-	projectID   = "test-project"
-	region      = "us-central1"
-	zoneName    = "us-central1-b"
-	nodeName    = "test-node-1"
-	clusterName = "Test Cluster Name"
-	clusterID   = "test-cluster-id"
-	serviceName = ""
+	projectID               = "test-project"
+	region                  = "us-central1"
+	zoneName                = "us-central1-b"
+	nodeName                = "test-node-1"
+	clusterName             = "Test Cluster Name"
+	clusterID               = "test-cluster-id"
+	serviceName             = "bar-svc"
+	eventReasonManualChange = "LoadBalancerManualChange"
+	eventMsgFirewallChange  = "Firewall change required by network admin"
+	errPrefixGetTargetPool  = "error getting load balancer's target pool:"
+	errStrLbNoHosts         = "Cannot EnsureLoadBalancer() with no hosts"
+	wrongTier               = "SupremeLuxury"
+	errStrUnsupportedTier   = "unsupported network tier: \"" + wrongTier + "\""
 )
 
-var apiService = &v1.Service{
-	Spec: v1.ServiceSpec{
-		SessionAffinity: v1.ServiceAffinityClientIP,
-		Type:            v1.ServiceTypeClusterIP,
-		Ports:           []v1.ServicePort{{Protocol: v1.ProtocolTCP, Port: int32(123)}},
-	},
+var apiService *v1.Service
+
+func setup() {
+	apiService = &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{Name: serviceName},
+		Spec: v1.ServiceSpec{
+			SessionAffinity: v1.ServiceAffinityClientIP,
+			Type:            v1.ServiceTypeClusterIP,
+			Ports:           []v1.ServicePort{{Protocol: v1.ProtocolTCP, Port: int32(123)}},
+		},
+	}
+}
+
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
```

Comment:
Humm..What happen if unit tests run parallelly? I was thinking just make:
``` golang
var apiService = v1.Service {...}
```
And everyone make a copy of it :)

or
```
func getTestService() *v1.Service {...}
```